Cargo.toml

```diff
@@ -35,6 +35,9 @@ sha1 = ["dep:sha1"]
 # openssl and sha1.
 wincrypto = ["dep:str0m-wincrypto"]
 
+# DTLS impl made for str0m (experimental) in pure Rust.
+dimpl = ["dep:dimpl"]
```

Comment:
Should the rust primitives only be added when dimpl is enabled?

Reply:
dimpl = ["dep:dimpl"], "dep:aes", ..... etc

Reply:
Yeah, good point. They're not really related, but for simplicity we maybe make them.